infra/storage/shared_file_system_managers/kSharedFileSystemMgr.php

```diff
@@ -636,4 +651,9 @@ public function copyDir($src, $dest, $deleteSrc)
 		return $this->doCopyDir($src, $dest, $deleteSrc);
 	}
 
+	public static function getCopySharedToSharedAllowed()
+	{
+		return self::$kCopySharedToShared;
```

Comment:
Better call getInstance form here and than ave the static member value based on what it returned.